src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_pe/PlayerAction.java

```diff
@@ -33,15 +33,19 @@ public void readFromClientData(ByteBuf clientdata, ProtocolVersion version) {
 	private static final int START_BREAK = 0;
 	private static final int ABORT_BREAK = 1;
 	private static final int STOP_BREAK = 2;
-	private static final int RELEASE_ITEM = 5;
-	private static final int STOP_SLEEPING = 6;
-	private static final int RESPAWN1 = 7;
+	//private static final int GET_UPDATED_BLOCK = 3;
+	private static final int RELEASE_ITEM = 4;
+	private static final int STOP_SLEEPING = 5;
+	private static final int RESPAWN1 = 6;
+	//private static final int JUMP = 7;
 	private static final int RESPAWN2 = 13;
```

Comment:
The respawn2 is actually DIMENSION_CHANGE_REQUEST, so maybe that id needs to be changed. Try dying in nether to see which one client sends when pressing respawn button.